test/modules/tilecache.js

```diff
@@ -19,10 +48,12 @@
             raiseEvent: function() {}
         };
         var fakeTiledImage0 = {
-            viewer: fakeViewer
+            viewer: fakeViewer,
+            source: tileSourceCacheAPI
```

Comment:
Could you use `TileSource.prototype` here instead?

Reply:
Oh should've thought about it, much cleaner.